README.md

```diff
@@ -49,48 +56,58 @@ $ tree ./opt-datadog-agent
 │   ├── mesos_master.d
 │   │   ├── conf.yaml
 │   │   └── private-key.pem
-│   └── mesos_slave.d
+│   ├── mesos_slave.d
+│   │   ├── conf.yaml
+│   │   └── private-key.pem
```

Comment:
maybe you can remove the `private-key.pem` from the repo.

Reply:
it's an empty key - I left it to make sure it's clear we need one, but it's a placeholder here.